### PR TITLE
Add the cwpair2 tool

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: packages/package_libxml2_2_9_3/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: package_libxml2_2_9_3
+owner: iuc
```
Comment (bgruening) [inline packages/package_libxml2_2_9_3/.shed.yml]: The package is here again ... strange. There will be some merge conflicts I guess.

File: packages/package_meme_4_11_0/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tool_dependency>
+    <package name="libxslt" version="1.1.28">
+        <repository name="package_libxslt_1_1_28" owner="iuc" prior_installation_required="True"/>
+    </package>
+    <package name="meme" version="4.11.0">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://meme-suite.org/meme-software/4.11.0/meme_4.11.0.tar.gz</action>
```
Comment (bgruening) [inline packages/package_meme_4_11_0/tool_dependencies.xml]: Please use cargo-port here as well.

File: tools/cwpair2/cwpair2.xml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                <option value="absolute_threshold">Absolute threshold</option>
+            </param>
+            <when value="relative_threshold">
+                <param  name="relative_threshold" type="float" value="0.0" min="0.0" label="Percentage of the 95 percentile value to filter below" help="Value 0 results in no filtering" />
+            </when>
+            <when value="absolute_threshold">
+                <param name="absolute_threshold" type="float" value="0.0" min="0.0" label="Absolute value to filter below" />
+            </when>
+        </conditional>
+        <param name="output_files" type="select" label="Restrict output to" help="Statistics will always be generated." >
```
Comment (bgruening) [inline tools/cwpair2/cwpair2.xml]: Can we change the logic here to say: "please select the output you want to have"

**Comment by bgruening** [comment]
Greg, this PR also contains the other one, correct? How do you want me to proceed with reviewing?

**Comment by gregvonkuster** [comment]
@bgruening I have no idea.  I am a git novice, so I'll take your directions on how to proceed.  I was using the approach of submitting 1 PR at a time and waiting until it got accepted before submitting a 2nd.  But @martenson advised me to submit them all in parallel as that would allow them to be accepted faster.

**Comment by bgruening** [comment]
Can you check which PR contains all tools and let me know. I will review this big one.

**Comment by martenson** [comment]
@gregvonkuster it is still the same idea of branches: you need to work on (and only on) the designated branch if you work on one tool. Take the branch as a context = If you switch tool/context you work on, you also switch the branch you work on. That will make it easy to maintain multiple contexts/tools/PR.

**Comment by gregvonkuster** [comment]
@martenson For each branch, I only worked on a specific tool of the same name as the branch, so I think I did that right.  So maybe where I went wrong was that I didn't check out the master branch when I created a new branch.  Can you confirm?

**Comment by martenson** [comment]
I think the issue is that your PR #478 is from `master` and you probably created all your new branches from master I assume. So all of the PRs contain the #478 PR in addition to your work on the tool in question.

**Comment by gregvonkuster** [comment]
Ok, so then I should not have submitted any additional PRs until the original PR on the master branch that contained the meme tool was merged?

**Comment by martenson** [comment]
Well that is one solution...; the better solution imo is to create proper branches from `galaxyproject:master` instead of your local unmerged master.

**Comment by martenson** [comment]
closing in favor of https://github.com/galaxyproject/tools-iuc/pull/498
please continue your discussions there

**Pull request closed without merging**